ansible/roles/docker/README.adoc

```diff
@@ -1,7 +1,59 @@
-== Role: Docker
+= Role: Docker
 
-Cluster target type: K8s
+:toc:left
+:doctype: article
+:source-highlighter: rouge
 
-=== Description
+Installs docker on host.
+
+== Description
+
+This role will install a Docker on a linux machine. It will install the following components:
```

Comment:
Can you detail what is `a Docker` ? Is it a docker client, docker daemon, ....

Reply:
In the following lines it is described the 2 components that are installed, docker engine and docker cli.